CHANGELOG.next.asciidoc

```diff
@@ -25,6 +25,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Metricbeat*
 
 - Add new option `OpMultiplyBuckets` to scale histogram buckets to avoid decimal points in final events {pull}10994[10994]
+- Change cloud.provider from ec2 to aws in add_cloud_metadata. {issue}10775[10775]
```

Comment:
Could you mention here that this is to align with ECS?

Reply:
done, thanks!